org.freedesktop.Sdk.json.in

```diff
@@ -957,7 +956,7 @@
         },
         {
             "name": "glib2",
-            "config-opts": ["--disable-gtk-doc",  "--with-pcre=system" ],
```

Comment:
I wonder if it would be slightly better to change `--disable-gtk-doc` to an explicit `--enable-gtk-doc`, to make sure modules always try to find and enable gtk-doc.